config/Config-kernel.in

```diff
@@ -1184,10 +1194,64 @@ config KERNEL_SECURITY_SELINUX_SID2STR_CACHE_SIZE
 	depends on KERNEL_SECURITY_SELINUX
 	default 256
 
+config KERNEL_SECURITY_APPARMOR
+	bool "AppArmor Support"
+	select KERNEL_AUDIT
+	select KERNEL_SECURITY
+	select KERNEL_SECURITYFS
+	select KERNEL_SECURITY_PATH
+	select KERNEL_SECURITY_NETWORK
+
+config KERNEL_SECURITY_APPARMOR_BOOTPARAM_VALUE
+	int "AppArmor boot parameter"
+	depends on KERNEL_SECURITY_APPARMOR
+	default 0
+	help
+	  This option sets the default value for the kernel parameter
+	  'apparmor', which allows AppArmor to be enabled or disabled
+	  at boot. If this option is set to 0 (zero), the AppArmor
+	  kernel parameter will default to 0, disabling AppArmor at
+	  boot. If this option is set to 1 (one), the AppArmor
+	  kernel parameter will default to 1, enabling AppArmor at
+	  boot.
+
+	  AppArmor and SELinux cannot be used simultaneously, choose
+	  only one of them as default.
+
+config KERNEL_DEFAULT_SECURITY_APPARMOR
```

Comment:
Since the LSM issue is fixed, we can remove this right?

Reply:
No, we can't, if you want it ti start without boot arguments. Again: if LSM would enable AppArmor on it's own, I would be having both, SELinux and AppArmor active at same time. LSM mostly is order of execution, If we remove or don't enable that parameter, only way to use apparmor would be adding "security=apparmor" to kernel's cmdline. And if you don't enable boot parameter (One that you should set 1) even that isn't then available.